ts-rs/tests/semver.rs

```diff
@@ -0,0 +1,19 @@
+#![allow(deprecated)]
+#![cfg(feature = "chrono-impl")]
```

Comment:
I think this is supposed to be `#![cfg(feature = "semver-impl")]` right?

Reply:
Yeah my bad